openpype/hosts/fusion/plugins/create/create_exr_saver.py

```diff
@@ -1,38 +1,51 @@
 import os
 
-from openpype.pipeline import (
-    LegacyCreator,
-    legacy_io
-)
+import qtawesome
+
 from openpype.hosts.fusion.api import (
     get_current_comp,
-    comp_lock_and_undo_chunk
+    comp_lock_and_undo_chunk,
+    remove_instance,
+    list_instances
 )
 
+from openpype.pipeline import (
+    legacy_io,
+    Creator,
+    CreatedInstance
+)
 
-class CreateOpenEXRSaver(LegacyCreator):
 
+class CreateOpenEXRSaver(Creator):
```

Comment:
This will likely be separated into a `FusionCreator` once it works and there's need for multiple savers - since this is the ONLY creator it's directly inheriting from `openpype.pipeline.Creator` for easy development.